exercises/practice/yacht/test/yacht_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule YachtTest do
+  use ExUnit.Case
+
+  @data Application.app_dir(:yacht, "priv/canonical-data.json")
+        |> File.read!()
+        |> Jason.decode!()
+
+  for test_case <- @data["cases"] do
+    test test_case["description"] do
+      assert Yacht.score(
+               unquote(test_case["input"]["category"]),
+               unquote(test_case["input"]["dice"])
+             ) == unquote(test_case["expected"])
+    end
+  end
+end
```

Comment:
Thanks for explaining the context around this and why this isn't a valid approach for the way the exercises work. I will definitely update this PR to include a test suite with the tests all explicitly written out so it will work as expected for students. I am interested in helping with that test file generation project, but I understand how it's separate and outside the scope of the exercises.